File: 019-Find the Maximum Value and its Index in the Array/FindtheMaximumValueanditsIndexintheArray.cpp
#include<iostream>
using namespace std;

void MaxInArray(int arr[], int length){
    int MaxVal = arr[0];
    int MaxIndex = 0;

    for(int i = 1; i < length; i++){
        if(arr[i] > MaxVal){
            MaxVal = arr[i];
            MaxIndex = i;
        }
    }
    cout<<"Maximum Value: "<<MaxVal<<endl;
    cout<<"Index of Maximum Value: "<<MaxIndex<<endl;
}   

int main(){
    int arr[] = {1, 3, 5,7, 9, 3, 2, 8, 10, 13};
    int length = sizeof(arr)/ sizeof(arr[0]);

    MaxInArray(arr, length);

    return 0;
}

File: 019-Find the Maximum Value and its Index in the Array/FindtheMaximumValueanditsIndexintheArray.py
def MaxInArray(arr):
    max_val = arr[0]
    max_index = 0

    for i in range(1, len(arr)):
        if arr[i] > max_val:
            max_val = arr[i]
            max_index = i

    print(max_val)
    print(max_index)

arr = [23, 45, 82, 27, 66, 12, 78, 13, 71, 86]
MaxInArray(arr)